poker-pals/src/App.vue

```diff
@@ -19,7 +22,8 @@ export default {
     // Poker,
     // Profile,
     // Reports,
-    //Tables
+    //Tables,
+    HelloWorld
```

Comment:
Is there a reason you want to render the HelloWorld component here? I'm assuming the HelloWorld component renders your tables page, but is there a reason it's being used here?